src/utils/UIconstants.ts

```diff
@@ -29,6 +29,8 @@ export const DialogsId = {
     ADD_DIRECTORY: 'add_directory',
     ADD_NEW_CONTINGENCY_LIST: 'add_new_contingency_list',
     ADD_NEW_FILTER: 'add_new_filter',
+    ADD_NEW_EXPERT_FILTER: 'add_new_expert_filter',
```

Comment:
Expert filter is an obsolete name. I think it should not be added in the code anymore (unless it causes some compatibility issues but when it comes to pure UI like here I don't think this is the case) : 
```suggestion
    ADD_NEW_CRITERIA_FILTER: 'add_new_criteria_filter',
```